### DISCUSSION TITLE
Issue with Incorrect Location Detection on GitHub Education Form

### DISCUSSION THREAD

**Discussion by sujal58** [community/community#145697] [GitHub Education]
### Hi everyone,

I am currently filling out the GitHub Education form from my college, but it is incorrectly detecting that I am far from my institution. This seems to be a technical issue, possibly related to IP or location settings.

Could you please assist me in resolving this so I can successfully complete my application?

Thank you for your support.

**Comment by davevad93** [comment]
Hi @sujal58 ,

It's possible that GitHub has the wrong info about your school's location, contact support by opening a ticket [here](https://support.github.com/contact/education), select `My selected school has incorrect or incomplete information e.g. domains or campus location` and explain your issue.
